### PR TITLE
Charting fix, mainly for the daily one. Negative values were off.

### PR DESCRIPTION
**Pull request by epigramx**
Charting fix, mainly for the daily one. Negative values were off.

It solves the issue: #310

I'm not familiar with this framework so another testing would help.

### CODE REVIEW

**Comment by xmatthias** [comment]
Interrestingly, this fix is now causing a different issue (notice the cut-off tops and bottoms - as this test had some rather high profits and losses)).

2021-04-10-111724_489x333_scrot

I guess the problem is with values very close to 0.
The issue shows all losses within a -0.002 range - so i guess the problem is the max for the red candles (which i set at -0.01 to avoid having a red line at 0.0 - but it's obviously causing some problems.

Probably the best is to leave the line with `absoluteMin` as is (otherwise we'll see these cutoffs) - and only adjust the min/max to 0.0 instead of the static offset of -0.01 / -0.00001.

**Comment by epigramx** [comment]
It seems cut off at the "2" and "-2", that I didn't change at all, It might be find to change those to something large.

File: src/components/charts/DailyChart.vue [hunk 1]
```diff
@@ -84,12 +84,13 @@ export default class DailyChart extends Vue {
...
-              max: -0.01,
-              min: this.absoluteMin - 2,
+              max: 0.0,
+              min: -2,
               color: 'red',
             },
             {
-              min: -0.00001,
+              min: 0.0,
+              max: 2,
```
Comment (xmatthias) [inline src/components/charts/DailyChart.vue:93]: well you actually did change these / add these ...

Comment (epigramx) [reply src/components/charts/DailyChart.vue:93]: True. I'll test now with the high limits removed. The TradesLog already didn't have those anyway.

Comment (xmatthias) [reply src/components/charts/DailyChart.vue:93]: the problem is, for this to actually show up as a problem you got to have some very high winning days (which honestly, i had in that test-database accidentally - along with some high losing days) :laughing:

Comment (epigramx) [reply src/components/charts/DailyChart.vue:93]: Yeah I noticed. It was pretty ludicrous to get 200% 😄 . I noticed removing the limits draws the lines but a thin solid black. I'll test using the absolute min/max functions in an intuitive manner as limits.

Comment (xmatthias) [reply src/components/charts/DailyChart.vue:93]: i think using "min" for the red, and not using max should work.
It's problably an oddity of echarts that's causing one to need both limits and not the other - but could be worse :laughing:

Comment (epigramx) [reply src/components/charts/DailyChart.vue]: Does the latest commit work for you? I noticed it had both min and max functions anyway so it made some sense and it still works here. I didn't see the same for the hourly though and I couldn't find a fast way to reproduce them for it.

Comment (xmatthias) [reply src/components/charts/DailyChart.vue]: i think this should work.
I don't expect anyone to have an average for one hour of above 200% :laughing: for one day however, a few longer trades can cause that ...

**Review by xmatthias — APPROVED** [review]
LGTM :+1: 

Thanks a lot!

**Pull request merged**